api/base/serializers.py

```diff
@@ -406,6 +426,16 @@ def to_representation(self, value):
         return ret
 
 
+class FileCommentRelationshipField(RelationshipField):
+    def __init_(self, **kwargs):
```

Comment:
If you're just calling super do you need to define this? Also, I don't think it's working because it's `__init_` instead of `__init__`.